index.html

```diff
@@ -0,0 +1,214 @@
+<!DOCTYPE html>
+<html>
```

Comment:
Don't forget to include the attribute of `lang="en"` which lets a search engine know what language the information is being displayed in.